packages/bot-web-ui/src/components/load-modal.jsx

```diff
@@ -204,94 +228,96 @@ const Local = ({
-    rerenderTabs,
-    should_rerender_tabs,
-    setActiveTabIndex,
-    toggleLoadModal,
-    is_mobile,
-    ...props
-}) => {
-    return is_mobile ? (
-        <FadeWrapper is_visible={is_load_modal_open} className='load__wrapper' keyname='save__wrapper'>
-            <PageOverlay header={localize('Load Strategy')} onClickClose={toggleLoadModal} onMount={onMount}>
-                <MobileWrapper>
-                    <Div100vhContainer className='load__wrapper--is-mobile' height_offset='80px'>
-                        <Tabs
-                            active_index={active_index}
-                            onTabItemClick={setActiveTabIndex}
-                            top
-                            fit_content
-                            header_fit_content
-                        >
-                            <div label={localize('Local')}>
-                                <Local is_mobile={is_mobile} {...props} />
-                            </div>
-                            <div label='Google Drive'>
-                                <GoogleDrive is_mobile={is_mobile} {...props} />
-                            </div>
-                        </Tabs>
-                    </Div100vhContainer>
-                </MobileWrapper>
-            </PageOverlay>
-        </FadeWrapper>
-    ) : (
+const LoadModal = props => {
+    const {
+        active_index,
+        is_load_modal_open,
+        is_mobile,
+        onEntered,
+        setActiveTabIndex,
+        should_rerender_tabs,
+        toggleLoadModal,
+    } = props;
+    const header_text = localize('Load strategy');
+
+    if (is_mobile) {
+        return (
+            <FadeWrapper is_visible={is_load_modal_open} className='load__wrapper' keyname='save__wrapper'>
+                <PageOverlay header={header_text} onClickClose={toggleLoadModal}>
+                    <MobileWrapper>
+                        <Div100vhContainer className='load__wrapper--is-mobile'>
+                            <Tabs
+                                active_index={active_index}
+                                onTabItemClick={setActiveTabIndex}
+                                should_delay_render={should_rerender_tabs}
+                                top
+                                fit_content
+                                header_fit_content
+                            >
+                                <div label={localize('Local')}>
+                                    <Local {...props} />
+                                </div>
+                                <div label='Google Drive'>
+                                    <GoogleDrive {...props} />
+                                </div>
+                            </Tabs>
+                        </Div100vhContainer>
+                    </MobileWrapper>
+                </PageOverlay>
+            </FadeWrapper>
+        );
+    }
+    return (
         <Modal
-            title={'Load Strategy'}
+            title={header_text}
```

Comment:
Just curious, why not localize the title here? 
e.g. `title={localize('Load Strategy')}`